.eslintrc.json

```
@@ -0,0 +1,52 @@
+{
+    "env": {
+        "node": true,
+        "mocha": true,
+        "es6": true
+    },
+    "parser": "@typescript-eslint/parser",
+    "extends": [
+        "plugin:@typescript-eslint/recommended",
+        "plugin:import/recommended",
+        "plugin:import/typescript"
+    ],
+    "rules": {
+        "indent": [
+            "error",
+            2
+        ],
+        "no-console": "off",
+        "import/no-unresolved": "off",
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        "quotes": [
+            "error",
+            "single"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ],
+        "eol-last": [
+            "error",
+            "always"
+        ],
+        "no-trailing-spaces": "error",
+        
+        /** Turn off strict enforcement */
```

```suggestion
        /** Turn off enforcement */
```

I think off turns off completely, not just make it "not strict